mesh/block.go

```diff
@@ -1,98 +1,137 @@
 package mesh
 
 import (
+	"bytes"
+	"fmt"
+	"github.com/davecgh/go-xdr/xdr2"
 	"github.com/google/uuid"
+	"github.com/spacemeshos/go-spacemesh/common"
+	"io"
 	"time"
 )
 
 type BlockID uint32
 type LayerID uint32
 
-var layerCounter LayerID = 0
-
+type Block struct {
+	Id          BlockID
+	LayerIndex  LayerID
+	Data        []byte
+	Coin        bool
+	Timestamp   time.Time
+	Txs         []SerializableTransaction
+	BlockVotes 	[]BlockID
```

Comment:
supposed to be a set of block ids for a specific layer

Reply:
the main point was that we dont include partition by layer id not the set
 also im not convinced that mesh's representation of block would be the one passing on the wire